src/components/Profile/MyPosts/Post/Post.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Consider adding an empty line between a third-party imports and your css.

Reply:
complite